step-55/step-55.cc

```diff
@@ -18,29 +18,22 @@
  */
 
 #include <deal.II/base/quadrature_lib.h>
+#include <deal.II/lac/block_linear_operator.h>
+#include <deal.II/lac/linear_operator.h>
 #include <deal.II/base/function.h>
 #include <deal.II/base/timer.h>
-
-// The following chunk out code is identical to step-40 and allows
-// switching between PETSc and Trilinos:
-
+#include <deal.II/lac/petsc_block_sparse_matrix.h>
 #include <deal.II/lac/generic_linear_algebra.h>
+#include <deal.II/lac/petsc_block_vector.h>
 
-/* #define FORCE_USE_OF_TRILINOS */
+// /* #define FORCE_USE_OF_TRILINOS */
```

Comment:
? I meant that you should use 
```suggestion
#define FORCE_USE_OF_TRILINOS
```
(that would mean you don't need the changes below)